data/human/hails.txt

```diff
@@ -1708,6 +1708,89 @@ phrase "friendly civilian"
 		", but I still had to take a big loan to buy my first ship."
 		", but it was my grandma's inheritance that allowed me to buy this ship."
 
+phrase "friendly civilian"
+	word
+		"Don't be too picky about the jobs you take. It's better to"
+	word
+		" "
+	word
+		"ship crooks, than having a crooked ship."
+		"ship slugs, than having a sluggish ship."
+		"ship bad bands, than having your ship badly banned."
+		"be working hard at shipping, than having your ship hardly working."
+
+
+phrase "friendly civilian"
+	word
+		"Ey Comrade,"
+	word
+		" "
+	word
+		"your ship has loaded on a planet? I have loaded a planet onto my ship."
+		"you have cargo in your ship? I have ships in my cargo."
+		"you have fighters in your bays? We have bays in our fighters."
+
+phrase "friendly civilian"
+	word
+		"Somebody"
+		"Some nut"
+	word
+		" "
+	word
+		"put"
+		"placed"
+	word
+		" "
+	word
+		"a blue phonebooth"
+		"a silver car"
+	word
+		" "
+	word
+		"on my bridge"
+		"in my armoury"
+		"in my cargohold"
+	word
+		" and "
+	word
+		"claimed to be"
+		"told my crew to be"
+	word
+		" "
+	word
+		"a time traveller"
+		"from another time"
+	word
+		", "
+	word
+		"and suddenly it disappeared."
+		"but the only travel he did was to a mental institution."
+		"I still don't know how she got into my spaceship."
+		"the thing looked bigger on the inside than on the outside."
+		"and some weirdos were flying around it on hoverboards."
```

Comment:
Line 1768 can result in hails like `Some nut put a silver car in my armoury and claimed to be a time traveller, I still don't know how she got into my spaceship.` which would be a run-on sentence. In addition, line 1769 would also create a run-on sentence, ex: `Some nut put a silver car in my armoury and claimed to be a time traveller, the thing looked bigger on the inside than on the outside`. 
```suggestion
	word
		", and suddenly it disappeared."
		", but the only travel he did was to a mental institution."
		". I still don't know how she got into my spaceship."
		". The thing looked bigger on the inside than on the outside."
		", and some weirdos were flying around it on hoverboards."
```
This would result in the sentence `Some nut put a silver car in my armoury and claimed to be a time traveller. I still don't know how she got into my spaceship.` or `Some nut put a silver car in my armoury and claimed to be a time traveller. The thing looked bigger on the inside than on the outside` which is grammatically correct.
Also, the line `and some weirdos were flying around it on hoverboards.` doesn't really make sense to me. Were there multiple people who got into the captain's ship? Where did they go? Apologies if this is a reference to something that I don't get.

Reply:
It would indeed be strange if a captain would allow some weirdos on hoverboards to just stay hanging around on his ship. I removed that option.

Reply:
I also applied your other review-comments in a separate commit.